src/gui/creds/shibbolethcredentials.cpp

```diff
@@ -129,9 +134,6 @@ bool ShibbolethCredentials::ready() const
 
 void ShibbolethCredentials::fetchFromKeychain()
 {
-    if (_user.isEmpty()) {
-        _user = _account->credentialSetting(QLatin1String(userC)).toString();
```

Comment:
The HTTP crendatials still does fetch the user from this point.  I think we should still be consistent with the HTTP credentials.  So perhaps you should keep it there.  Or remove it as well from the HTTP credentials.